config/locales/client.en.yml

```diff
@@ -1181,6 +1181,10 @@ en:
         categories_section_instruction: "Selected categories will be displayed under Sidebar's categories section."
         tags_section: "Tags Section"
         tags_section_instruction: "Selected tags will be displayed under Sidebar's tags section."
+        navigation_section: "Navigation"
+        topic_destination_instruction: "When I click a topic list link in the sidebar with new or unread topics, take me to"
+        topic_destination_latest: "Latest"
+        topic_destination_unread: "New/Unread"
```

Comment:
this name is not really idea. `topic_destination_unread_new` which is a mouthful is probably more correct, that is the intent here.